mlir/lib/Dialect/Linalg/IR/LinalgInterfaces.cpp

```diff
@@ -222,7 +222,7 @@ bool mlir::linalg::detail::isContractionBody(
   Value contributed = getSourceSkipUnary(
       isa<BlockArgument>(reductionLHS) ? reductionRHS : reductionLHS);
   Operation *elementwiseOp = contributed.getDefiningOp();
-  if (elementwiseOp->getNumResults() != 1 ||
+  if (!elementwiseOp || elementwiseOp->getNumResults() != 1 ||
```

Comment:
Can you please provide a minimal test for this?

Reply:
Hi~ Added a test to the existing test method. This test can reproduce the problem of the current method.

Reply:
Hello~   Could you please help me to approve this PR?  Thanks~